README.md

```diff
@@ -12,56 +12,112 @@
+## Features
+
+<p>
+  <ul style="font-size: 18px">
+    <li>Support for Enterprise Integration Patterns</li>
+    <li>Create Camel Routes, Kamelets and Pipes</li>
+    <li>Supports OpenAPI to REST DSL code generation</li>
+    <li>Support for Beans</li>
+    <li>300+ Camel Components</li>
+    <li>200+ Kamelets</li>
+    <li>Built-in catalog of Components, Kamelets and EIPs with documentation</li>
+    <li>Run your integrations locally</li>
+  </ul>
+</p><br/>
 
-### Features
+## Installation
 
-- Edit Camel files following pattern (`*.camel.yaml` and `*.camel.yml`).
-- Edit Kamelet files following pattern (`*.kamelet.yaml` and `*.kamelet.yml`).
-- Edit Pipe files following pattern (`*.pipe.yaml`, `*.pipe.yml`, `*-pipe.yaml` and `*-pipe.yml`)
-- Allow to edit `*.yaml` and `*.yml` when opening through contextual menu.
-- Edit Kaoto (`*.kaoto.yaml` and `*.kaoto.yml`) files. Note that these filename extensions are not supported by Camel JBang and are merely still here to stay backward-compatible with Kaoto v1.
 
-### Limitations
+### Requirements
 
-- Kaoto files are always written and overwritten with `Linux-style` end of line (EOL).
+<p>
+  <ul style="font-size: 18px">
+    <li>Microsoft VS Code installed. Get the latest version at <a href="https://code.visualstudio.com/">https://code.visualstudio.com/</a>.</li>
+    <li>Camel CLI installed. Read the <a href="https://camel.apache.org/manual/camel-jbang.html">installation instructions</a>.</li>
+  </ul>
+</p><br/>
 
-### Embedded
+### Install the VS Code extension
+<p align="left" style="font-size: 18px">
+<ul style="font-size: 18px">
+<li>Open VS Code</li>
+<li>Open the Extensions view on the left side panel (or press <code>CTRL+SHIFT+X</code>)</li>
+<li>Type <code>Kaoto</code> in the search field</li>
+<li>Click the <code>Install</code> button</li>
+</ul>
+<img style="width: 30%; height: auto;" src="./images/kaoto-install.png" />
+</p><br/>
 
-- [Kaoto 2 release](https://github.com/KaotoIO/kaoto) in version [2.0.0](https://github.com/KaotoIO/kaoto/releases/tag/2.0.0).
+### Create your integration
+<p align="left" style="font-size: 18px">
+<ul style="font-size: 18px">
+<li>Open the command palette by pressing <code>F1</code> or <code>CTRL+SHIFT+P</code></li>
+<li>Type <code>camel</code> in the search field</li>
+<li>Select what you want to do</li>
+</ul>
+<img style="width: 30%; height: auto;" src="./images/kaoto-create.png" />
+</p><br/>
 
-### Issues
+### Run your integration
+<p align="left" style="font-size: 18px">
+<img style="width: 30%; height: auto;" src="./images/kaoto-run.png" />
+</p><br/>
 
-Something is not working properly? In that case, feel free to [open issues, add feature requests, report bugs, etc.](https://github.com/KaotoIO/vscode-kaoto/issues).
+### Rapid Prototyping
+[![A video showing how to do rapid prototyping with Kaoto and Camel JBang](https://img.youtube.com/vi/sLuBa5aoWyc/0.jpg)](https://www.youtube.com/watch?v=sLuBa5aoWyc)
```

Comment:
There is a better integration with VS Code debug adapter than this clipboard thingy which is less convenient and less secure; i think we should not point VS Code users to it.